hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/HederaNode.java

```diff
@@ -121,7 +128,8 @@ CompletableFuture<Void> statusFuture(
      * @return this node's HAPI spec identifier
      */
     default String hapiSpecInfo() {
-        return getHost() + ":" + getGrpcPort() + ":0.0." + getAccountId().accountNumOrThrow();
+        return getHost() + ":" + getGrpcPort() + ":" + getGrpcNodeOperatorPort() + ":0.0."
```

Comment:
I would revert this change (and the changes to `NodeConnectInfo` below).

Reply:
Reverted.